### DISCUSSION TITLE
windows上怎么使用？

### DISCUSSION THREAD

**Discussion by banxiayueguang** [SeaHOH/GotoX#27] [问答/咨询]
阅读之后请删除这些说明文字，然后写下你的问题，谢谢。

提问之前，你可以浏览一下本项目首页的简单说明。
描述清楚问题的情况下标题尽量简洁；
可能的话附上你的系统信息和使用状况；
附上出错时的日志，如果能确认范围就尽量不要全部复制过来，或者截图也可以；
贴文字日志时建议使用以下格式：

```
日志开始
……
日志结束
```

**Comment by jzp820927** [comment]
提示让你装 gevent，你就装 gevent 就行了。

**Comment by horizonl** [comment]
pip install gevent

**Comment by banxiayueguang** [comment]
![pyopensll](https://cloud.githubusercontent.com/assets/27133152/25424311/87a8da06-2a9a-11e7-8ba1-05256d550fa7.png)

**Comment by banxiayueguang** [comment]
如上图怎么办

**Comment by jzp820927** [comment]
你是不是在 Windows 上自己安装了 python？

**Comment by banxiayueguang** [comment]
@jzp820927 是的

**Comment by jzp820927** [comment]
不在 cmd 运行命令，直接运行 GotoX.vbs 试试

**Comment by banxiayueguang** [comment]
闪退

**Comment by SeaHOH** [comment]
@banxiayueguang 
从那个位置下载的？
* 主分支源码，需自行安装依赖。
    * `pip install gevent`
    * `pip install pyopenssl`
* release，直接运行 GotoX.vbs。

**Comment by banxiayueguang** [comment]
release 在哪下载？

**Comment by SeaHOH** [comment]
仓库主页上面……

**Comment by banxiayueguang** [comment]
是不是"Clone or download"里面的download zip

**Comment by SeaHOH** [comment]
只能出动绝招了，你狠。 :trollface: 
![_2017-04-26t09-00-28 638z](https://cloud.githubusercontent.com/assets/8470820/25426485/e68bc324-2aa1-11e7-90a6-a981dcf8f96e.png)

**Comment by banxiayueguang** [comment]
是啊 毕竟小白 多谢大佬指导
